Tools/ardupilotwaf/boards.py

```diff
@@ -238,6 +238,10 @@ def configure_env(self, cfg, env):
             env.LIB += [
                 'winmm',
             ]
+            env.AP_LIBRARIES += [
```

Comment:
because there is a common dependency in dataflash I think, I am surprised that only cygwing are failing ...
But this file is for board config not librairies, for libraries it depend on vehicule wscript